showcase/app/components/shw/autoscrollable/index.gts

```diff
@@ -58,4 +58,14 @@ export default class ShwAutoscrollable extends Component<ShwAutoscrollableSignat
       verticalShift: this.args.verticalShift ?? 0,
     });
   });
+
+  <template>
+    <div
+      class="shw-autoscrollable__container"
+      ...attributes
+      {{this.autoscroll}}
```

Comment:
Note: I tried to move the autoscroll modifier definition outside of the class/Component declaration, but it didn't work: for some reasons that I don't understand only the last instance actually called the `centerScrollableArea` function.

If you know why and want to give it a go and try do it "properly" feel free to push directly in the branch.

Reply:
I did attempt to do the same but gave up as this modifier is closely tide to this components. It might be good to follow up as a stand alone PR in the future